acceptance/tests/utf8/utf8-in-puppet-describe.rb

```diff
@@ -0,0 +1,80 @@
+platforms = hosts.map {|val| val[:platform]}
+if (platforms.any? { |val| /^eos-/ =~ val})
+  skip_test "Skipping because Puppet describe fails when the Arista module is installed (ARISTA-51)"
+end 
+
+test_name 'utf-8 characters in module doc string, puppet describe' do
+  # utf8chars = "€‰ㄘ万竹ÜÖ"
+  utf8chars = "\u20ac\u2030\u3118\u4e07\u7af9\u00dc\u00d6"
+  agents.each do |agent|
+    puts "agent name: #{agent.hostname}, platform: #{agent.platform}"
```

Comment:
I was testing with a hosts.yaml file that specified ~40 different agent types.  A similar test failed on a small subset of them.  Adding that output made it easier to tell which platforms had passed, and how far along through the set of agents the test had come, and which platform failed.  I guess it is not that important here.  Should I remove it?